src/main/resources/application.properties

```diff
@@ -3,12 +3,12 @@ spring.datasource.url=jdbc:h2:mem:testdb
 spring.datasource.driverClassName=org.h2.Driver
```

Comment:
You can add `spring.config.import=optional:file:.env[.properties]` and take env var from .env file

Reply:
Done